provider/resources.go

```diff
@@ -2649,6 +2651,8 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_data_factory_credential_user_managed_identity": {
 				Docs: &tfbridge.DocInfo{AllowMissing: true},
 			},
+			"azurerm_chaos_studio_target":      {Tok: azureResource(azureChaosStudio, "Target")},
+			"azurerm_redhat_openshift_cluster": {Tok: azureResource(azureRedHatOpenShift, "Cluster")},
```

Comment:
The mappings look correct here. Instead of adding manual mappings, can we add an auto-mapping to [`moduleMap`](https://github.com/pulumi/pulumi-azure/blob/6c88b7f781911804ee33c08791020982af1ea6d7/provider/resources.go#L187)? That way we won't need to intervene when the next chaos_studio resource or datasource is released.

```go
"chaos_studio":     azureChaosStudio,
"redhat_openshift": azureRedHatOpenShift,
```